content/docs/observable.md

```diff
@@ -0,0 +1,202 @@
+# Observable
+
+> [tip box yellow] This functionality was added in v2.0.0-alpha.202 and is still experimental.
+
+### TLDR
+Imba provides MobX-like reactivity implemented as decorators.
+- Mark class and tag members as observable with the `@observable` decorator.
+- Methods decorated with `@autorun` will be reactively invoked whenever
+referenced observables change.
+- Getters decorated with `@computed` will be cached on first use
+and only recomputed whenever referenced observables change.
+
+See also [additional details](#additional-details).
```

Comment:
I don't think we need "see also" for something that appears so nearby on the page.